src/packages.builds

```diff
@@ -10,7 +16,7 @@
     <Project Include="Native\pkg\**\*.builds">
       <AdditionalProperties>$(AdditionalProperties);SkipCreatePackageOnMissingFiles=true</AdditionalProperties>
     </Project>
-    <Project Include="*\pkg\*.builds">
+    <Project Include="*\pkg\*.builds" Condition="'$(FilterToOSGroup)' == 'Windows_NT'">
```

Comment:
We should provide an additional optional property to allow folks to disable this and not filter this to only be windows like this.